lib/Doctrine/DBAL/Connection.php

```diff
@@ -912,10 +919,10 @@ public function executeQuery(string $query, array $params = [], $types = [], ?Qu
      */
     public function executeCacheQuery($query, $params, $types, QueryCacheProfile $qcp) : ResultStatement
     {
-        $resultCache = $qcp->getResultCacheDriver() ?? $this->_config->getResultCacheImpl();
+        $resultCache = $qcp->getResultCacheDriver() ?: $this->_config->getResultCacheImpl();
```

Comment:
@jwage is this intentional? Most likely, it comes from an incorrect conflict resolution.

Reply:
Yes, bad conflict resolution. I will fix, one second.